pkgs/top-level/all-packages.nix

```diff
@@ -28546,6 +28546,7 @@ with pkgs;
   inherit (callPackage ../misc/uboot {})
     buildUBoot
     ubootTools
+    ubootRockchipTools
```

Comment:
If we were to use that, we probably wouldn't want to expose it as a package, as this is the kind of package that should be removed ASAP as it is an unfortunate one to have to maintain.